packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -162,12 +166,14 @@ export const announceTheme = async ({
   designSystem: any
   runtimeConfiguration?: RuntimeConfiguration
 }): Promise<void> => {
```

Comment:
Loading the theme must also happen before any UI is rendered, if we puth this request in the background UI flickering can happen.

Reply:
right now yes but we have plans to do the login page rendering different. Then this can conflict.

Reply:
Why? As I said, my implementation keeps waiting for `announceTheme` before finishing `bootstrap`. So nothing changes in the behavior of `bootstrap`. If nothing changes, if you have conflicts with what I did, you will have conflicts with what you already have...

One thing that you really need to do in the login workflow is to redirect the user to the idp before trying to load the full application (which takes ages).